hack/update-openapi-spec.sh

```diff
@@ -50,33 +50,33 @@ kube::golang::setup_env
 TMP_DIR=$(mktemp -d /tmp/update-openapi-spec.XXXX)
 ETCD_HOST=${ETCD_HOST:-127.0.0.1}
 ETCD_PORT=${ETCD_PORT:-2379}
-API_PORT=${API_PORT:-8050}
+API_PORT=${API_PORT:-6443}
```

Comment:
why was this changed?

Reply:
I think 6443 indicates this is a secure port, I could revert this change if you want.

Reply:
yeah, I think the goal was to pick a port that would not conflict with an existing running server